dev/docker/hive/hive-dependency.sh

```diff
@@ -10,13 +10,17 @@ hive_dir="$(cd "${hive_dir}">/dev/null; pwd)"
 # Environment variables definition
 HADOOP_VERSION="2.7.3"
 HIVE_VERSION="2.3.9"
+MYSQL_JDBC_DRIVER_VERSION="8.0.15"
 
 HADOOP_PACKAGE_NAME="hadoop-${HADOOP_VERSION}.tar.gz" # Must export this variable for Dockerfile
 HADOOP_DOWNLOAD_URL="http://archive.apache.org/dist/hadoop/core/hadoop-${HADOOP_VERSION}/${HADOOP_PACKAGE_NAME}"
 
 HIVE_PACKAGE_NAME="apache-hive-${HIVE_VERSION}-bin.tar.gz" # Must export this variable for Dockerfile
 HIVE_DOWNLOAD_URL="https://archive.apache.org/dist/hive/hive-${HIVE_VERSION}/${HIVE_PACKAGE_NAME}"
 
+JDBC_DIVER_PACKAGE_NAME="mysql-connector-java-${MYSQL_JDBC_DRIVER_VERSION}.tar.gz" # Must export this variable for Dockerfile
+JDBC_DIVER_DOWNLOAD_URL="https://downloads.mysql.com/archives/get/p/3/file/${JDBC_DIVER_PACKAGE_NAME}"
```

Comment:
Why do you use this version? We used another version in JDBC integration tests and playground.
https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.27/mysql-connector-java-8.0.27.jar

Reply:
We have been using this version from the beginning(see following ref) in Hive-CI-Docker, but the JDBC integration tests and the playground introduced later are inconsistent with the previous version.

https://github.com/datastrato/gravitino/blob/dbdc1c9a6cbf31cff9d56c2e93a8bf8c1a9b13cd/dev/docker/hive/Dockerfile#L131-L132

Reply:
OK, we don't need to keep consistent in this pr. I guess we can change them to a consistent version in later pr.